storybook/tailwind.scss

```diff
@@ -0,0 +1 @@
+@tailwind utilities;
```

Comment:
We could also add `@tailwind base;` here - tailwinds reset for native browser styles but I've held off for the moment in case we want to advise differently for our consumers

Reply:
You can see the issues with not providing a reset here:
![Screen Shot 2022-11-03 at 12 08 56 pm](https://user-images.githubusercontent.com/36558508/199630135-98cc7196-5d6e-470d-be47-c9fd24a49cef.png)